apps/console/src/features/identity-providers/api/identity-provider.ts

```diff
@@ -1222,7 +1223,7 @@ export const updateOutboundProvisioningConnectors = <T = Record<string,unknown>>
 /**
  * Update a federated authenticators list of a specified IDP.
  *
- * @param authenticatorList - 
+ * @param authenticatorList -
```

Comment:
Add a description

Reply:
Resolved in https://github.com/wso2/identity-apps/pull/3699/commits/1ce362734978a7fc5bb714b1a67a03ea0db9ae79